src/components/seeds/database/columns.ts

```diff
@@ -197,12 +204,28 @@ export const defaultPreset = (system?: string): Preset => {
 
 const generalInventoryPreset: Preset = {
   name: 'General Inventory',
-  fields: ['accessionNumber', 'speciesName', 'collectionSiteName', 'collectionSiteLandowner', 'state', 'collectedDate'],
+  fields: [
+    'accessionNumber',
+    'speciesName',
+    'project_name',
+    'collectionSiteName',
+    'collectionSiteLandowner',
+    'state',
+    'collectedDate',
+  ],
 };
 
 const seedStoragePreset: Preset = {
   name: 'Seed Storage Status',
-  fields: ['accessionNumber', 'speciesName', 'state', 'collectedDate', 'facility_name', 'subLocation_name'],
+  fields: [
+    'accessionNumber',
+    'speciesName',
+    'project_name',
+    'state',
+    'collectedDate',
+    'facility_name',
+    'subLocation_name',
+  ],
```

Comment:
probably want the feature flag guard in all places in this file.. the columns edit config modal uses these to display column names

<img width="411" alt="Terraware App 2023-12-13 14-17-00" src="https://github.com/terraware/terraware-web/assets/1865174/77d53c6c-14e9-402c-a5b3-edc9b685bd84">

Reply:
This is the same as https://github.com/terraware/terraware-web/pull/1901#discussion_r1425937210, fixed in latest commit